### PR TITLE
Mint token on select hub / open channel view

### PR DESCRIPTION
**Pull request by nephix**
Implements #1071 token minting on select hub / open channel view

Screenshot 2020-02-26 at 09 07 45

Screenshot 2020-02-26 at 11 17 19

Screenshot 2020-02-26 at 11 17 09

### CODE REVIEW

**Comment by christianbrb** [comment]
@nephix Nice :) Very good idea to add the screen shots 👍 

Could you also update the [Try out guide within the readme](https://github.com/raiden-network/light-client#try-out-the-raiden-demo-dapp)

**Comment by christianbrb** [comment]
Readme looks great now :)

**Comment by nephix** [comment]
If one of the reviewers can please merge (squash & merge) as well that'd be nice, as codecov seems to hang again

File: raiden-dapp/src/locales/en.json [hunk 1]
```diff
@@ -342,6 +347,11 @@
       "problem": "We do not know this error, sorry.",
       "solution": "Please file a new issue on https://github.com/raiden-network/light-client."
     },
+    "RDN_MINT_FAILED": {
+      "title": "Could not mint token",
+      "problem": "It looks like the Light Client does not support minting out-of-the-box for this token contract.",
+      "solution": "You may try to mint manually via MetaMask and Etherscan. You can find a step-by-step guide on https://github.com/raiden-network/light-client#minting-manually."
```
Comment (andrevmatos) [inline raiden-dapp/src/locales/en.json]: Nitpick: couldn't this be converted in the link properly? Having the address as [non-clickable] text is kind of ugly

Comment (nephix) [reply raiden-dapp/src/locales/en.json]: No it can't otherwise I would have done it

**Pull request merged**